code/modules/mob/living/simple_animal/hostile/wumborian_fugu.dm

```diff
@@ -137,9 +137,9 @@
 		A.buffed++
 		A.maxHealth *= 1.5
 		A.health = min(A.maxHealth,A.health*1.5)
-		A.melee_damage = max((A.melee_damage * 2), 10)
+		A.melee_damage = max((A.melee_damage * 1.5), 10)
+		A.obj_damage = max(A.melee_damage, A.obj_damage) //Some mobs already have turbo object damage, and this doesn't need to be boosted. This ensures it's never less than their basic damage though.
 		A.transform *= 2
-		A.environment_smash |= ENVIRONMENT_SMASH_STRUCTURES | ENVIRONMENT_SMASH_RWALLS
```

Comment:
Isn't this just an AI flag which lets them know they are capable of breaking walls?

Reply:
Alright yeah, this entire PR was born of me being lied to and mis-interpreting the code afterwards. 
Not anything wrong with fugu glands, though I do still think the 100% boost in damage is a bit steep.